src/util/config.rs

```diff
@@ -167,6 +273,182 @@ pub fn parse_store_labels(labels: &str) -> Result<HashMap<String, String>, Confi
     Ok(map)
 }
 
+pub struct ReadableSize(pub u64);
```

Comment:
1.5GB will be serialized as "1536MB", and "1.5GB" will be deserialized as 1610612736.